README.md

```diff
@@ -116,6 +117,42 @@ shiny-server: 1.3.0.403
 
 ----------
 
+## Create and enable swap file
+
+-   This step is necessary when running on a T2.mirco EC2 instance. You won't be able to install certain R packages (e.g. `dplyr`, `tidyr`) without the additional allocated memory from the swap 
```

Comment:
If you can make this section stand out more (e.g. bold or italics) that would be great!  I'm assuming an audience of very limited background when they are reading this guide.

Overall, well-written and simple to follow, thanks for this!